core/src/main/java/org/mskcc/cbio/portal/oncoPrintSpecLanguage/GeneticTypeLevel.java

```diff
@@ -39,8 +39,8 @@
  */
 public enum GeneticTypeLevel{
     // this order must be maintained for DiscreteDataTypeSpec.satisfy 
-    HomozygouslyDeleted (-2, GeneticDataTypes.CopyNumberAlteration, "Homdel"),
```

Comment:
Ok, actually, let's not make any changes to stuff in `oncoPrintSpecLanguage`. I'm pretty sure it's only used by the old JSPs. AFAIK, they're in a weird limbo where we don't maintain them, but a few people still use them. Better not to touch them. Maybe @jjgao can confirm?

Reply:
Reverted change to file